docs/install/kubernetes.md

```diff
@@ -57,11 +57,10 @@ to log in and manage templates.
    [Postgres operator](https://github.com/zalando/postgres-operator) to
    manage PostgreSQL deployments on your Kubernetes cluster.
 
-1. Download the latest `coder_helm` package from
-   [GitHub releases](https://github.com/coder/coder/releases).
+1. Add the Coder Helm repo:
 
    ```console
-   wget https://github.com/coder/coder/releases/download/<release>/coder_helm_<release>.tgz
+   helm repo add coder https://helm.coder.com/v2
    ```
```

Comment:
We shouldn't tell people to use the same name for the helm repo otherwise it'll cause problems if they ever tried Coder v1.

Reply:
That's a good point but considering that most people will be trying coder for the first time what do you think about simply adding an extra line indicating to call it something else if you are coming from v1?